### DISCUSSION TITLE
Universal Control

### DISCUSSION THREAD

**Discussion by ghost** [devboloji/Infinix-Hackintosh-Opencore-Guide#22] [Q&A] [answered]
I was wonder if it was possible to use the universal control feature on this hackintosh device?
I tried to use it and nothing happens with timed out message.

Infinix INBook X1 Pro

**Comment by devboloji** [comment, marked as answer]
@bunnybavros I don't tried yet brother and I don't want to fix it for the future tooo.. It's waste of time.

**Reply by ghost** [reply]
Yes, it is not working.
I have tried to use it with my iPad M1, and every time I tried to use it, it would give me a message that the request was timed out.

I might try to fix it myself, just to make sure that it is possible.

**Reply by devboloji** [reply]
@bunnybavros Let's think about it later. 😉

**Reply by ghost** [reply]
@devboloji thank you about that anyway! make sure to not overdo for yourself man. ✌
